serde/src/de/impls.rs

```diff
@@ -396,125 +396,108 @@ impl<T> Deserialize for PhantomData<T> {
 
 macro_rules! seq_impl {
     (
-        $ty:ty,
-        $visitor_ty:ident < $($typaram:ident : $bound1:ident $(+ $bound2:ident)*),* >,
-        $visitor:ident,
-        $ctor:expr,
-        $with_capacity:expr,
-        $insert:expr
+        $ty:ident< $($typaram:ident : $bound1:ident $(+ $bound2:ident)*),* >,
+        $with_capacity:expr
     ) => {
-        /// A visitor that produces a sequence.
-        pub struct $visitor_ty<$($typaram),*> {
```

Comment:
[This is a breaking change.](https://github.com/search?l=Rust&q=%22impls%3A%3AVecVisitor%22&ref=searchresults&type=Code&utf8=%E2%9C%93)